src/Common/ErrorPopup/ErrorPopup.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { ReactElement, useEffect, useContext } from "react";
+import CloseIcon from "../../../public/assets/icons/CloseIcon";
+import styles from "./ErrorPopup.module.scss";
+import { useTranslation } from "next-i18next";
+import { QueryParamContext } from "src/Layout/QueryParamContext";
+import { SerializedError } from "@planet-sdk/common";
+import { ThemeContext } from "styles/themeContext";
+
+export default function ErrorPopup(): ReactElement {
+  const { t, ready } = useTranslation(["common"]);
+  const { errors, setErrors } = useContext(QueryParamContext);
+  const { theme } = useContext(ThemeContext);
+
+  useEffect(() => {
+    let timer: NodeJS.Timeout;
+    if (errors) {
+      timer = setTimeout(() => {
+        setErrors(null);
+      }, 5000);
```

Comment:
1. Should we keep a timeout at all? 
2. If we keep a timeout, should this be longer?

Reply:
1. If we don't keep timeout user will have to click the `x` icon to clear the errors. We can avoid user having to do extra work with setTimeout.

2. Most of the [error messages](https://github.com/Plant-for-the-Planet-org/planet-donations/blob/9d9a0602026d1833e42a1d716009dcb5edd4bf0e/public/locales/en/common.json#L179) are quite small and can be read under 5 seconds.

Reply:
The issue is that a disappearing error message makes it difficult for the user to report the error, or understand if they can do something to avoid the error. Such error messages could end up being missed. 

@mariahosfeld @sagararyal What do you think?

Reply:
Resolved - We've set a longer timeout (10 seconds)